modules/tabs-v2/assets/js/frontend/handlers/tabs-v2.js

```diff
@@ -0,0 +1,44 @@
+import BaseTabsV2 from 'elementor-frontend/handlers/base-tabs-v2';
+
+export default class TabsV2 extends BaseTabsV2 {
+	getDefaultSettings() {
```

Comment:
Try to move to the base

Reply:
Handled

Reply:
I think it can stay like this for a while tell we find out the best approach